File: src/cs/PhantasyIslandPythonRemoteControl/ImageProcess.cs
using System;

namespace PhantasyIslandPythonRemoteControl
{
    public static class ImageProcess
    {
        public static byte[] ReadB64Img(string uri)
        {
            if (string.IsNullOrEmpty(uri)) return null;
            try
            {
                var parts = uri.Split(',');
                if (parts.Length < 2) return null;
                return Convert.FromBase64String(parts[1]);
            }
            catch
            {
                return null;
            }
        }
    }
}

File: src/fs/ImageProcess.fs
namespace PhantasyIslandRemoteControl

open System

module ImageProcess =
    let readB64Img (uri: string) =
        if String.IsNullOrEmpty uri then null
        else
            try
                let parts = uri.Split(',')
                if parts.Length < 2 then null
                else Convert.FromBase64String(parts.[1])
            with _ -> null
